adal/src/main/java/com/microsoft/aad/adal/AuthenticationResult.java

```diff
@@ -302,6 +342,33 @@ public String getIdToken() {
         return mIdToken;
     }
 
+    /**
+     * Gets if the returned token is valid in terms of extended lifetime
+     *
+     * @return True if the returned token is valid in terms of extended lifetime
+     */
+    final public boolean isExtendedLifeTimeToken() {
+        return mIsExtendedLifeTimeToken;
+    }
+
+    /**
+     * Sets the property isExtendedLifeTimeToken to indicate whether the token
+     * being returned is a token only valid in terms of extended lifetime
+     *
+     * @param isExtendedLifeTimeToken
+     */
+    final void setIsExtendedLifeTimeToken(final boolean isExtendedLifeTimeToken) {
+        this.mIsExtendedLifeTimeToken = isExtendedLifeTimeToken;
```

Comment:
@addev-ashish, what's your opinion here? should we do a get/set or what Heidi currently has?